src/components/dashboard/SalesByHourChart.tsx

```
@@ -0,0 +1,177 @@
+
+  // Ensure we have all 24 hours (0-23)
+  const chartData = Array.from({ length: 24 }, (_, hour) => {
+    const found = data.find(d => d.hour === hour)
+    return found || { hour, sales: 0, percentage: 0 }
+  }).map(item => ({
+    ...item,
+    hourLabel: `${item.hour.toString().padStart(2, '0')}:00`,
+  }))
+
+  // Custom label component for percentages (only show if > 1%)
+  const renderCustomLabel = (props: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)